GSServer.py

```diff
@@ -37,22 +37,15 @@ def start_server(args, m=MVelKeepConfig()):
         btree_locations = [base_btree_location]
     log.info ("Btree search locations set (in order) as: " + str(btree_locations))
 
+    traffic = SimTraffic(lanelet_map, sim_config)
+
     if args.verify_map != "":
         verify_map_file(args.verify_map, lanelet_map)
         return
 
     if args.no_dash:
         sim_config.show_dashboard = False
 
-    if args.wait_for_input:
```

Comment:
why is this wait for input/client removed? Also I moved traffic creation after these input to line 53, see the comment.